content/oci/oci_test.go

```diff
@@ -1088,7 +1162,7 @@ func TestStore_ExistingStore(t *testing.T) {
 	}
 
 	// test resolving index root by tag
-	gotDesc, err := anotherS.Resolve(ctx, tag)
+	gotDesc, err := anotherS.Resolve(ctx, string(indexRoot.Digest))
```

Comment:
Is it doing the same thing as line 1174?

Reply:
Yes, you are right. I've updated the tests in order for all the checks to be relevant again.